sys/psa_crypto/psa_crypto.c

```diff
@@ -1133,6 +1133,15 @@ static psa_status_t psa_start_key_creation(psa_key_creation_method_t method,
     slot = *p_slot;
     slot->attr = *attributes;
 
+    /* See 9.5.2. Key usage flags */
+    if (slot->attr.policy.usage | PSA_KEY_USAGE_SIGN_HASH) {
+        slot->attr.policy.usage |= PSA_KEY_USAGE_SIGN_MESSAGE;
```

Comment:
```suggestion
    if (slot->attr.policy.usage & PSA_KEY_USAGE_SIGN_HASH) {
        slot->attr.policy.usage |= PSA_KEY_USAGE_SIGN_MESSAGE;
```

same below. Can you double-check there is no PSA API or RIOT implementation convenience function that tests for a specific usage flag?

Reply:
I will.

Reply:
Thanks for checking. Please squash :)